### ISSUE TITLE
how to get ade20k imgs?

### ISSUE THREAD

**Issue by Finger-tiao** [praeclarumjj3/Experiments-CoCosNet#1]
I download the ADE20K dataset but get label image with different colors, not the gray images like your "ade20k/img" file.
<img width="1380" alt="屏幕快照 2021-05-14 上午12 28 40" src="https://user-images.githubusercontent.com/32674239/118156726-4569ae00-b44c-11eb-86ae-82c211035ec4.png">

Do I need to do anything with the origin ADE20K dataset?

**Comment by praeclarumjj3** [comment]
Which file are you referring to here? And what do you mean by "gray images"?

No, you don't need to do anything with the downloaded ADE20K dataset. Which script did you execute? @Finger-tiao

**Comment by Finger-tiao** [comment]
Thanks for your reply. Sorry I'm slow to response.
I followed the official CoCosNet code to train it on ADE20K dataset. But the error is shown as below:
<img width="1534" alt="屏幕快照 2021-05-17 下午8 52 28" src="https://user-images.githubusercontent.com/32674239/118492468-9f22ee80-b752-11eb-838e-4ed321ebd9b1.png">

It's seem like the label is cross-border and it's not the dataset's fault based on your reply. but I have no ideas.  I saw you also did the experiment of CoCosNet using ADE20K dataset so do you have any idea about this error?  Thanks.

**Comment by praeclarumjj3** [comment]
I see you are trying to train the network. I didn't do anything with the training. I only ran test experiments using the pre-trained models provided. 

However, looking at your errors, it seems an issue with this line:

```
input_semantics = input_label.scatter_(1, label_map, 1.0)
```

It might be helpful for you to look at the shapes of `label_map` and `input_label` to ensure if the index argument of the [scatter_()](https://pytorch.org/docs/master/generated/torch.Tensor.scatter_.html#:~:text=Writes%20all%20values%20from%20the,in%20index%20for%20dimension%20%3D%20dim%20.) function is passed incorrectly or not.

**Comment by praeclarumjj3** [comment]
I am closing this issue since this repo only focuses on test setting experiments. Feel free to open an issue in the official CoCosNet repo if you still face any issues :)
